src/components/pages/rules/rulesGrid/rulesGrid.js

```diff
@@ -32,19 +35,35 @@ export class RulesGrid extends Component {
 
     // TODO: This is a temporary example implementation. Remove with a better version
     this.contextBtns = {
-      disable: <Btn key="disable">Disable</Btn>,
+      disable: <Btn key="disable" svg={svgs.disableToggle} onClick={this.openDisableFlyout}>{props.t('rules.flyouts.disable')}</Btn>,
+      enable: <Btn key="enable" svg={svgs.enableToggle} onClick={this.openEnableFlyout}>{props.t('rules.flyouts.enable')}</Btn>,
+      changeStatus: <Btn key="changeStatus" svg={svgs.loadingToggle} onClick={this.openChangeStatusFlyout}>{props.t('rules.flyouts.changeStatus')}</Btn>,
       edit: <Btn key="edit" svg={svgs.edit} onClick={this.openEditRuleFlyout}>{props.t('rules.flyouts.edit')}</Btn>
     };
   }
 
   openEditRuleFlyout = () => this.setState({ openFlyoutName: 'edit' });
 
+  openDisableFlyout = () => this.setState({ openFlyoutName: 'disable' });
+
+  openEnableFlyout = () => this.setState({ openFlyoutName: 'enable' });
+
+  openChangeStatusFlyout = () => this.setState({ openFlyoutName: 'changeStatus' });
+
   setSelectedRule = selectedRule => this.setState({ selectedRule });
 
+  setSelectedRules = selectedRules => this.setState({ selectedRules });
```

Comment:
There is no meaningful distinction between rule and rules in this case. Especially considering that the flyout itself is converting the the singular rule back into an array, which you already have here. Just pass the selected rules as an array and use the length for any important plurality distinctions.